Whatsapp_Automated_Messages/__main__.py

```diff
@@ -0,0 +1,103 @@
+import pyautogui as pt 
+import time
+import pyperclip
+import random 
+
+time.sleep(4)
+
+position1 = pt.locateOnScreen("Whatsapp_Automated_Messsages/smilie_paperclip.png", confidence=.6)
+x= position1[0]
+y= position1[1]
+
+
+def get_message():             # it will help us to fetch the messages
+    global x, y
+
+    position = pt.locateOnScreen("./Whatsapp_Automated_Messsages/smilie_paperclip.png", confidence=.6)
+    x=position[0]
```

Comment:
this x, y is also repeated throughout the code. Until these values are not changing, there is no need to define them again and again